frontend/pages/login/logo/index.js

```diff
@@ -0,0 +1 @@
+export { Logo } from "./logo";
```

Comment:
Why we need a `logo/index.js` and a `logo/logo.js`? What the diff between import the logo/index and logo/logo file?

Reply:
`logo/index.js` is for simplify imports like `import logo from './logo'` instead `import logo from './logo/logo`.  We can move the logo source for `index` file and reduce the number of files, but, an problem with this approach is that, when editing multiple Components at once in your editor, the tabs are all labeled index.js.
What you think about?

Reply:
Maybe we can reduce the number of files for now, until this becomes a problem.

Reply:
No problem about that just was trying to understand the `why` behind this approach. Your explanation makes all sense, for me is good to go this way (with index files for import and component.file with real code).

About the error when trying to run the project; I use the same command that I was used to `docker-compose up`. :eyes: